src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -174,16 +178,18 @@ class CalculatorForm extends React.Component {
                 label="Degree Level"
                 name="yellowRibbonDegreeLevel"
                 alt="Degree Level"
+                hideArrows={yellowRibbonDegreeLevelOptions.length <= 1}
                 options={yellowRibbonDegreeLevelOptions}
-                visible
+                visible={showYellowRibbonOptions}
```

Comment:
Hide the drop downs when there is yellow ribbon support but no specific programs to select from.